Cargo.lock

```diff
@@ -610,7 +610,7 @@ dependencies = [
 [[package]]
 name = "kvproto"
 version = "0.0.1"
-source = "git+https://github.com/pingcap/kvproto.git#1b5d69cd1d263f9473f7ed2847a7660af394c437"
+source = "git+https://github.com/pingcap/kvproto.git#0ba3ca8a6e3775accbe4bdf844162cd7685dafe2"
```

Comment:
The master branch now uses a newer revision of kvproto (updated by another PR yesterday), and your changes to kvproto is already available in TiKV's master. Please keep this line unchanged, then your PR can be built.